test/versioned/langchain/bedrock/runnables.test.js

```diff
@@ -0,0 +1,450 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const test = require('node:test')
+const assert = require('node:assert')
+
+const { removeModules } = require('../../../lib/cache-buster')
+const { assertPackageMetrics, assertSegments, assertSpanKind } = require('../../../lib/custom-assertions')
+const {
+  assertLangChainChatCompletionMessages,
+  assertLangChainChatCompletionSummary,
+  filterLangchainEvents,
+  filterLangchainEventsByType
+} = require('../common')
+const { version: pkgVersion } = require('@langchain/core/package.json')
+const { getAiResponseServer } = require('../../aws-sdk-v3/common')
```

Comment:
this is going to fail because it loads `@aws-sdk/client-bedrock-runtime` from the aws versioned tests